clients/src/test/java/org/apache/kafka/common/security/oauthbearer/internals/secured/AccessTokenRetrieverFactoryTest.java

```diff
@@ -55,21 +63,34 @@ public void testConfigureRefreshingFileAccessTokenRetriever() throws Exception {
     @Test
     public void testConfigureRefreshingFileAccessTokenRetrieverWithInvalidDirectory() {
         // Should fail because the parent path doesn't exist.
-        Map<String, ?> configs = getSaslConfigs(SASL_OAUTHBEARER_TOKEN_ENDPOINT_URL, new File("/tmp/this-directory-does-not-exist/foo.json").toURI().toString());
+        String file = new File("/tmp/this-directory-does-not-exist/foo.json").toURI().toString();
+        System.setProperty(ALLOWED_SASL_OAUTHBEARER_URLS_CONFIG, file);
+        Map<String, ?> configs = getSaslConfigs(SASL_OAUTHBEARER_TOKEN_ENDPOINT_URL, file);
         Map<String, Object> jaasConfig = Collections.emptyMap();
         assertThrowsWithMessage(ConfigException.class, () -> AccessTokenRetrieverFactory.create(configs, jaasConfig), "that doesn't exist");
     }
 
     @Test
     public void testConfigureRefreshingFileAccessTokenRetrieverWithInvalidFile() throws Exception {
-        // Should fail because the while the parent path exists, the file itself doesn't.
+        // Should fail because the parent path exists, the file itself doesn't.
```

Comment:
If think it should be `// Should fail because while the parent path exists, ...`

Reply:
You're right!